library/cartridge_check_instance_started.py

```diff
@@ -8,21 +8,40 @@
 argument_spec = {
     'control_sock': {'required': True, 'type': 'str'},
     'stateboard': {'required': True, 'type': 'bool'},
+    'waiting_for_state': {'required': False, 'type': 'list', 'default': ['Unconfigured', 'RolesConfigured']},
 }
 
 
 def check_instance_started(params):
+    waiting_for_state = params['waiting_for_state']
     try:
         control_console = get_control_console(params['control_sock'])
         ok = True
         if not params['stateboard']:
             ok = control_console.eval('''
                 return require('membership').myself().status == 'alive'
             ''')
+            if ok is True:
```

Comment:
Redundant `if`, I think

Reply:
Probably, yes. And return in case `ok == False` shouldn't be `Instance is not running`. Is `unhealthy` a membership status as well? in that case it will fail but instance IS running, not healthy though